docs/docsite/rst/dev_guide/style_guide/spelling_word_choice.rst

```diff
@@ -271,7 +271,7 @@ When referring to the reader, use "you" instead of "user." For example, "The use
 
 Username
 ^^^^^^^^
-Correct. Do not use "user name."
+Correct. Do not use "username."
```

Comment:
remove this - similarly this is where we tell people not to use user name.  And again, we want to keep this PR focused on fixing filename.